acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/web3j/privacy/PrivacyGroupAcceptanceTest.java

```diff
@@ -131,4 +192,27 @@ public void nodeCanCreatePrivacyGroupWithoutOptionalParams() {
     assertThat(privacyGroups.get(0).getDescription()).isEqualTo("Default Description");
     assertThat(privacyGroups.get(0).getMembers().length).isEqualTo(2);
   }
+
+  private void verifyGroupWasCreated() {
+    privacyNet
+        .getNode("Alice")
```

Comment:
should 'name' be a parameter? I.e. thus you can prove the privacygroup was created on any/all nodes in the network?

Reply:
addressed in #1864